lib/chef/mixin/which.rb

```diff
@@ -19,6 +19,10 @@ class Chef
   module Mixin
     module Which
       def which(cmd, opts = {})
+        Chef::Mixin::Which.which(cmd, opts)
+      end
+
+      def self.which(cmd, opts = {})
```

Comment:
doesn't this not work without the `included?` hook?  and can't you `extend` vs `include` to mix in the module methods into the class instead of the instance?